packages/eslint-config-kibana/.eslintrc.js

```diff
@@ -1,183 +1,129 @@
-const semver = require('semver');
+const APACHE_2_0_LICENSE_HEADER = `
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+`;
 
-const PKG = require('../../package.json');
-const RESTRICTED_GLOBALS = require('./restricted_globals');
+const ELASTIC_LICENSE_HEADER = `
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+`;
 
 module.exports = {
-  parser: 'babel-eslint',
-
-  plugins: [
-    'mocha',
-    'babel',
-    'react',
-    'react-hooks',
-    'import',
-    'no-unsanitized',
-    'prefer-object-spread',
-    'jsx-a11y',
+  extends: [
+    './javascript.js',
+    './typescript.js',
+    './jest.js',
   ],
+  plugins: ['@kbn/eslint-plugin-eslint'],
 
-  settings: {
-    react: {
-      version: semver.valid(semver.coerce(PKG.dependencies.react)),
+  overrides: [
+    /**
+     * Prettier
+     */
+    {
+      files: [
+        '.eslintrc.js',
+        'packages/kbn-eslint-plugin-eslint/**/*',
+        'packages/kbn-config-schema/**/*',
+        'packages/kbn-pm/**/*',
+        'packages/kbn-es/**/*',
+        'packages/elastic-datemath/**/*',
+        'packages/kbn-i18n/**/*',
+        'packages/kbn-dev-utils/**/*',
+        'packages/kbn-plugin-helpers/**/*',
+        'packages/kbn-plugin-generator/**/*',
+        'packages/kbn-test-subj-selector/**/*',
+        'packages/kbn-test/**/*',
+        'packages/kbn-eslint-import-resolver-kibana/**/*',
+        'src/legacy/server/saved_objects/**/*',
+        'x-pack/plugins/apm/**/*',
+        'x-pack/plugins/canvas/**/*',
+        '**/*.{ts,tsx}',
+      ],
+      plugins: ['prettier'],
+      rules: Object.assign(
+        {
+          'prettier/prettier': ['error'],
+        },
+        require('eslint-config-prettier').rules,
```

Comment:
I think you can remove these manual `require`s and `plugins` with `extends`:

```js
extends: [
  'plugin:prettier/recommended',
  'prettier/react'
],
rules: {
  'prettier/prettier': ['error']
}
```

Reply:
@eliperelman yes we could however overriding the `extends` is not possible yet (https://github.com/eslint/rfcs/pull/13). So until it become available I would like to keep it like this so we can override those rules in other configurations (javascript, typescript, jest...) if we want too 😃

Reply:
@mistic I'm curious as to why prettier isn't being applied at the top level like `javascript.js`, `typescript.js`, and `jest.js` extensions are.